lib/compat/wordpress-5.9/theme.json

```diff
@@ -215,8 +210,8 @@
 					"size": "36px"
 				},
 				{
-					"name": "Huge",
-					"slug": "huge",
+					"name": "x-large",
```

Comment:
Name should be "Extra Large"

Reply:
Updated https://github.com/WordPress/gutenberg/pull/37381/commits/3860b83bb91b06229da01acfb84de48d4af5d6d4